File: week6/absencap.java
import java.util.Scanner;
class AccessSpecifierDemo{
   private int privar;
   protected int provar;
   public int pubvar;

   public void setvar(int privalue, int provalue, int pubvalue)
   {
       this.privar = privalue;
       this.provar = provalue;
       this.pubvar = pubvalue;
   }

   public void getvar()
   {
       System.out.println("Private Variable : " +privar);
       System.out.println("Protected Variable : " +provar);
       System.out.println("Public Variable : " +pubvar);
   }
      
}  
public class absencap{
   public static void main(String[] args) {
       AccessSpecifierDemo obj = new AccessSpecifierDemo();
       Scanner input = new Scanner(System.in);
       int privalue, provalue, pubvalue;
       System.out.print("Enter the value of Private Variable : ");
       privalue = input.nextInt();
       System.out.print("Enter the value of Protected Variable : ");
       provalue = input.nextInt();
       System.out.print("Enter the value of Public Variable : ");
       pubvalue = input.nextInt();
       obj.setvar(privalue, provalue, pubvalue);
       obj.getvar();
       input.close();

   }
}

File: week6/absencap.cpp
#include<iostream>
using namespace std;

class AccessSpecifierDemo{
   private:
       int privar;
   protected:
       int provar;
   public:
       int pubvar;

   public:
       void setvar(int privalue, int provalue, int pubvalue)
       {
           this->privar = privalue;
           this->provar = provalue;
           this->pubvar = pubvalue;
       }

       void getvar(){
           cout << "Private Variable : " << privar << endl;
           cout << "Protected Variable : " << provar << endl;
           cout << "Public Variable : " << pubvar << endl;
       }

};

int main(){
   AccessSpecifierDemo obj;
   int privalue, provalue, pubvalue;
   cout << "Enter the value of Private Variable : ";
   cin >> privalue;
   cout << "Enter the value of Protected Variable : ";
   cin >> provalue;
   cout << "Enter the value of Public Variable : ";
   cin >> pubvalue;
   obj.setvar(privalue, provalue, pubvalue);
   obj.getvar();
}
